sopel/modules/admin.py

```diff
@@ -160,9 +189,11 @@ def temporary_part(bot, trigger):
 def restart(bot, trigger):
     """Restart the bot. This is an owner-only command."""
     quit_message = trigger.group(2)
+    default_message = 'Restart on command from %s.' % trigger.nick
```

Comment:
The old message was grammatically incorrect for _how long_? \*doesn't check git blame because it's probably depressing\*

```suggestion
    default_message = 'Restarting on command from %s.' % trigger.nick
```